_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,153 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each target element:
+
+- is categorized to follow the [flow content model][], and;
+- that it's contents are its [descendants][] in the DOM, and;
+- has at least one node in its contents that is a [palpable content][], which does not have the `hidden` attribute specified.
```

Comment:
I don't understand why "palpable content" is needed. It is not part of the content model of either `dl`, `ol`, or `ul`.

Reply:
For comment above and this [comment](), the spec only refers to usage of the hidden attribute & not other ways of hiding the content. See spec - https://html.spec.whatwg.org/#palpable-content